prep-javascript-for-loops/main.js

```diff
@@ -0,0 +1,20 @@
+// create your loops here.
+var counter = 0;
+var numbers = [];
+for (let i = 0; i < 10; i++) {
+  counter =
+    numbers.push(counter);
```

Comment:
Hi Simon, rather than pushing each number into an array, you can print it right here with `console.log()`